### PR TITLE
Add send nft method

### PR DESCRIPTION
**Pull request by DiegoCerdaC**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -666,16 +692,16 @@ class WebSDK {
     );
   };
 
-  openPinCode = (chargeId: string) => {
+  openPinCode = (target: string = 'SEND_NFT') => {
```
Comment (matzapatasphereone) [inline index.ts:695]: It would be nice here to use an enum instead that we can keep up to date with our server.

**Review by matzapatasphereone — APPROVED** [review]
LGTM

**Review by DavidYangSphere1 — APPROVED** [review]
@DiegoCerdaC , looks fine to me.

File: index.ts (hunk 1 above)
Comment (DavidYangSphere1) [reply index.ts]: For now, we can keep this as a string. Cause if we do the enums, it will be kinda conflicting for normal chargeId, which is a string.

**Pull request merged**